Library/Formula/efl.rb

```diff
@@ -0,0 +1,49 @@
+class Efl < Formula
+  homepage "https://www.enlightenment.org"
+  url "http://download.enlightenment.org/rel/libs/efl/efl-1.14.0-beta1.tar.gz"
+  sha256 "567d96849cc13be2eaf899d21c9902492a2c3aca111bcd0e91adabe65c4af6bd"
+
+  conflicts_with "eina", :because => "efl aggregates formerly distinct libs, one of which is eina"
+  conflicts_with "evas", :because => "efl aggregates formerly distinct libs, one of which is evas"
+  conflicts_with "eet", :because => "efl aggregates formerly distinct libs, one of which is eet"
+  conflicts_with "embryo", :because => "efl aggregates formerly distinct libs, one of which is embryo"
+
+  option "with-docs", "Install development libraries/headers and HTML docs"
+
+  depends_on "pkg-config" => :build
+  needs :cxx11 # Workaround for Mountain Lion
```

Comment:
Done. Since you wrote "as well" (wrt the comment), I removed the other comment about Mountain Lion as well. It's in a separate commit though, so you can drop it if you didn't meant for it to be removed too.